src/utils.js

```diff
@@ -21,7 +21,10 @@ const setupCml = async opts => {
   if (sudo) {
     try {
       sudoPath = await exec('which sudo');
-    } catch (err) {}
+    } catch (err) {
+      console.log('Failed find the sudo binary.', err);
+      process.exit(1);
```

Comment:
@0x2b3bfa0 I tested this to review the error logging and added the `exit 1`. In this particular example, without the `process.exit()`, the rest of the code can actually run to complete without error since `sudoPath` results in a blank string.

You can see a test [here](https://github.com/DanielBarnes/errrrrr/runs/3690231307?check_suite_focus=true)